src/handlers/mentions.rs

```diff
@@ -50,6 +51,19 @@ pub(super) async fn parse_input(
         return Ok(None);
     }
 
+    // Don't ping if a bors commit is included - send a warning message instead
+    if event
+        .issue
+        .commits(&ctx.github)
+        .await
+        .map_err(|e| log::error!("failed to fetch commits: {:?}", e))
+        .unwrap_or_default()
+        .into_iter()
+        .any(|commit| commit.commit.author.name == "bors")
+    {
+        return Ok(Some(MentionsInput::HasBorsCommit));
```

Comment:
Yeah, seems like this isn't the most unreasonable behavior for that case though. The PR author could of course ping manually, but I also don't think that it's even necessary in case of a subtree bump

Reply:
It does seem just as relevant to ping on a subtree bump if it touches code outside the subtree (as is not uncommon, I believe, though maybe not particularly desirable).

Reply:
Oh, I did not know that was possible. How does that happen?

I'll update in that case

Reply:
https://github.com/rust-lang/rust/issues/101907#issuecomment-1287936891 would also help here -- it would mean we could distinguish between Miri-bors commits and rustc-bors commits.